package.json

```diff
@@ -47,6 +47,7 @@
     "onCommand:truffle-vscode.createLocalUI",
     "onCommand:truffle-vscode.newSolidityProject",
     "onCommand:truffle-vscode.buildContracts",
+    "onCommand:truffle-vscode.buildSingleContract",
```

Comment:
Instead of having two different commands to achieve similar results, we could have one command as before, that act on files or folders depending on where it was invoked.

Reply:
@acuarica, Yes, i've tried to keep only one command. But i did not find any documentation explaining how to change the command title for 2 entries. Do you know how to do that?